docs: add trg 2-4 leading readme example (#551)
eclipse-tractusx/eclipse-tractusx.github.io: docs/release/trg-2/trg-2-4.md

title: TRG 2.04 - Leading product repository
---

| Status | Created     | Post-History                                   |
|--------|-------------|------------------------------------------------|
| Update | 12-Dez-2023 | add example leading repository reference links |
| Active | 07-Mar-2023 |                                                |
|        | 10-Jan-2023 | Initial release                                |

## Why

> We have multiple stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.

Reviewer: external people? from what?  how about ..., interested persons trying it out, ...

Author: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

They need to be able to understand complicated product structures in an intuitive way to have an easier task of diving into a product.

## Description

Each product **must** have a leading product repository that points to all the parts/components/charts/releases of the product.
This can be achieved by having a single repository for a product or creating references to the other repositories.

Exemplary repository structure:

- productname: leading product repository

>   - README.md: contains the urls for e.g. the backend and frontend applications

Reviewer: urls -> URLs  (btw, multiple times in the Text)

Author: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

>   - README.md: each repo follows our [TRG 1.01](../trg-1/trg-1-1.md) about the README.md

Reviewer: has to follow TRG 1.01 about README.md?

Author: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

title: TRG 2.04 - Leading product repository
---

| Status | Created     | Post-History                                   |
|--------|-------------|------------------------------------------------|
| Update | 12-Dez-2023 | add example leading repository reference links |
| Active | 07-Mar-2023 |                                                |
|        | 10-Jan-2023 | Initial release                                |

## Why

> We have multiply stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.

Reviewer 2: multiple instead of multiply 😁 was already wrong in the original

Author: [6d1bdb2](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/6d1bdb2ddb0f607fae53a875d68eb018db5809be)
done

They need to be able to understand complicated product structures in an intuitive way to have an easier task of diving into a product.

## Description

Each product **must** have a leading product repository that points to all the parts/components/charts/releases of the product.
This can be achieved by having a single repository for a product or creating references to the other repositories.

> Good examples of products got separate repositories for each component (e.g. frontend and backend).

Reviewer 2: Not sure, what this sentence should tell me. I think we don't lose much, if we keep this one out.

Reviewer 3: i think the examples are missing here?

> The product **must** have a leading repository where they reference the components/repositories of the product:

Reviewer 2: Duplicate to the first sentence in "## Description"

Author: [6d1bdb2](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/6d1bdb2ddb0f607fae53a875d68eb018db5809be)

title: TRG 2.04 - Leading product repository
---

| Status | Created     | Post-History                                   |
|--------|-------------|------------------------------------------------|
| Update | 12-Dez-2023 | add example leading repository reference links |
| Active | 07-Mar-2023 |                                                |
|        | 10-Jan-2023 | Initial release                                |

## Why

> We have multiple stakeholders for a repository: system team, release management, test management, interested persons trying it out, want to contribute or need to set it up.

Reviewer: ```suggestion
We have multiple stakeholders for a repository: system team, test management, release management and everybody else interested in the product.
```
